nd4j/samediff-import/samediff-import-onnx/src/main/kotlin/org/nd4j/samediff/frameworkimport/onnx/definitions/OnnxOpDeclarations.kt

```diff
@@ -360,8 +360,18 @@ val gemm = OnnxMappingProcess(
                 invertBooleanNumber(mutableMapOf("transX" to "transA","transY" to "transB")))
 )
 //TODO: GlobalAveragePool
+val globalAveragePooling = OnnxMappingProcess(
+        opName = "noop",
+        inputFrameworkOpName = "GlobalAveragePool",
+        opMappingRegistry = onnxOpRegistry
+)
 //TODO: GlobalLpPool
 //TODO: GlobalMaxPool
+val globalMaxPooling = OnnxMappingProcess(
+        opName = "noop",
+        inputFrameworkOpName = "GlobalMaxPool",
+        opMappingRegistry = onnxOpRegistry
+)
```

Comment:
Is this done? If yes, remove the TODO markers, if not, add a note what is missing.

Reply:
Updated and clarified context around op.